src/components/SearchItem/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { AiOutlineRight } from 'react-icons/ai/index';
+import * as Styles from './styles';
+
+import { useWeather } from '../../hooks/WeatherHook/index';
+
+interface SearchItemProps {
+	results: Record<any, any>[];
```

Comment:
Humm... would be awesome if we could know what is this result type. I think that we can replace it to some object like `{ title: string,  woeid: number }` or something else with the same idea.